src/main/java/com/documaster/rms/noark/ws/serialization/NoarkOptionalEnumJsonSerializer.java

```diff
@@ -0,0 +1,20 @@
+package com.documaster.rms.noark.ws.serialization;
+
+import java.io.IOException;
+import java.util.Optional;
+
+import com.documaster.rms.noark.ws.constants.NoarkEnum;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+
+public class NoarkOptionalEnumJsonSerializer extends JsonSerializer<Optional<NoarkEnum>> {
+
+	@Override
+	public void serialize(
+			Optional<NoarkEnum> saksstatus, JsonGenerator jsonGenerator, SerializerProvider serializerProvider)
```

Comment:
`saksstatus` should probably be renamed to `noarkEnum`

Reply:
Fixed with [188562f](https://github.com/documaster/ws-client-n5-java/pull/34/commits/188562f55c147739d63683eaab0adc477cff709c)